test/formatCurrency.spec.js

```diff
@@ -0,0 +1,24 @@
+import { expect } from "chai";
+import React from "react";
+import { shallow } from "enzyme";
+import Globalize from "globalize";
+import FormatCurrency from "../dist/currency";
+
+Globalize.load(
+    require( "cldr-data/main/en/numbers" ),
+    require( "cldr-data/main/en/currencies" ),
+    require( "cldr-data/supplemental/currencyData" ),
+    require( "cldr-data/supplemental/likelySubtags" )
+);
+
+describe("formatCurrency Component", () => {
+    it("renders as a <span>", () => {
+        const wrapper = shallow(<FormatCurrency locale="en" currency="USD">{150}</FormatCurrency>);

```

Comment:
Similarly, we can set the default locale as English with `Globalize.locale( "en" )` in a preliminary `before` step for all tests and get rid of this attribute in all tests.

Perhaps one test could assert the optional `locale` attribute to ensure it overrides the default one.